benefits/static/css/styles.css

```
@@ -574,6 +575,21 @@ footer .footer-links li a.footer-link:visited {
   gap: 1.5rem;
 }
 
+/* Forms: Select Inputs */
+/* Styles specific to the agency chooser on the landing page */
+
```

Compensating for not being wrapped in a `<p>` anymore, and adding a bit of line-height:


```suggestion

#form-agency-selection label {
  font-size: var(--font-size-20px);
  line-height: var(--bs-body-line-height);
  letter-spacing: calc(var(--font-size-20px) * var(--letter-spacing-2));
}

```